src/components/atoms/cellTable/cellTable.scss

```
@@ -0,0 +1,31 @@
+.sensorCell {
```

```suggestion
.sensor-cell {
```
nom de class sous cette forme non ?!